lib/block-supports/index.php

```diff
@@ -37,16 +38,26 @@ function gutenberg_apply_block_supports( $block_content, $block ) {
 
 	$block_type = WP_Block_Type_Registry::get_instance()->get_registered( $block['blockName'] );
 	// If no render_callback, assume styles have been previously handled.
-	if ( ! $block_type || ! $block_type->render_callback ) {
+	// Custom CSS is also a server-side hook even for static blocks.
+	$has_custom_css_support = false;
+	if ( property_exists( $block_type, 'supports' ) ) {
+		$has_custom_css_support = gutenberg_experimental_get( $block_type->supports, array( 'customCSS' ), false );
+	}
```

Comment:
Ideally, this shouldn't be necessary here and block support application should be "opt-in" maybe? or  use a  dedicated function like proposed by @mcsf

Reply:
There are a few ways to interpret this, and I think you and I discussed more than one way to handle it. Can you detail your suggestions?

Reply:
I'm just saying that the function you proposed `wp_block_supports` should make this useless once implemented but I guess it will force us to have a `render_callback` for the group block.

Reply:
Thinking more here, it seems the "automatic" approach we have now make this easier, otherwise we'll have to "duplicate" the group "save" function on the backend somehow?

Reply:
I'm super undecided here. I feel like this is probably not the solution we want for the future, but I feel out of good ideas. That's also why I stressed further below that we should keep the feature experimental. That way I'm not keeping your PR from being merged. :)